packages/loot-core/src/server/budget/types/templates.ts

```diff
@@ -4,14 +4,14 @@ interface BaseTemplate {
   directive: string;
 }
 
-interface PercentageTemplate extends BaseTemplate {
+export interface PercentageTemplate extends BaseTemplate {
```

Comment:
Can we move this to the types/models folder? e.g. types/models/templates.ts

Reply:
Sure. See 75a14a5e2792cfed62e00b03b599b7adf6bb7c7c